src/test/datascience/notebook/helper.ts

```diff
@@ -416,6 +416,26 @@ export async function stopJupyterServer() {
     await JupyterServer.instance.dispose().catch(noop);
 }
 
+let workedAroundVSCodeNotebookStartPage = false;
+/**
+ * VS Code displays a start page when opening notebooks for the first time.
+ * This takes focus from the notebook, hence our tests can fail as a result of this.
+ * Solution, try to trigger the display of the start page displayed before starting the tests.
+ */
+export async function workAroundVSCodeNotebookStartPages() {
+    if (workedAroundVSCodeNotebookStartPage) {
+        return;
+    }
+    workedAroundVSCodeNotebookStartPage = true;
+    const { editorProvider } = await getServices();
+    await closeActiveWindows();
+
+    // Open a notebook, VS Code will open the start page (wait for 5s for VSCode to react & open it)
+    await editorProvider.createNew();
+    await sleep(5_000);
```

Comment:
Blech. Every test takes 5 seconds longer? I guess it's only temporary. 

I'm wondering if there's a way to just wait for the editor open to fire?

Reply:
The only way is to see if we first have an active notebook editor, and then we don't have an 
 active notebook editor.
I've filed an issue to remove this hack (will be easier once webview/custom goes away).

Reply:
FYI https://github.com/microsoft/vscode-jupyter/issues/6113
Yes, all of this sux...

Reply:
My hack was to see if we had a visibleEditor, but no active editor, then showNotebookDocument to force it back to showing. That didn't require a wait.